tests/integration/test_crons.py

```diff
@@ -1,7 +1,7 @@
 def generate_check_in():
     return {
         "check_in_id": "a460c25ff2554577b920fcfacae4e5eb",
-        "monitor_id": "4dc8556e039245c7bd569f8cf513ea42",
+        "monitor_id": "monitor-id",
```

Comment:
```suggestion
        "monitor_slug": "monitor-id",
```
Same in the assertion below.